robottelo/cli/contenthost.py

```diff
@@ -53,8 +47,57 @@ def errata_info(cls, options):
             cls._construct_command(options), output_format='csv')
 
     @classmethod
-    def errata_apply(cls, options):
-        """Schedule errata for installation"""
-        cls.command_sub = 'errata apply'
+    def errata_list(cls, options):
+        """List errata available for the content host."""
+        cls.command_sub = 'errata list'
+        return cls.execute(
+            cls._construct_command(options), output_format='csv')
+
+    @classmethod
+    def package_install(cls, options):
+        """Install packages remotely."""
+        cls.command_sub = 'package install'
+        return cls.execute(
+            cls._construct_command(options), output_format='csv')
+
+    @classmethod
+    def package_remove(cls, options):
+        """Uninstall packages remotely."""
+        cls.command_sub = 'package remove'
+        return cls.execute(
+            cls._construct_command(options), output_format='csv')
+
+    @classmethod
+    def package_upgrade(cls, options):
+        """Update packages remotely."""
+        cls.command_sub = 'package upgrade'
+        return cls.execute(
+            cls._construct_command(options), output_format='csv')
+
+    @classmethod
+    def package_upgrade_all(cls, options):
+        """Update packages remotely."""
```

Comment:
Maybe 'Update all system packages remotely'? Otherwise the docstring looks exactly like what you have for the `package_upgrade` method